src/tools/demo.ts

```diff
@@ -6,6 +6,8 @@ export enum CommentType {
   REMOVE_BLOCK_END = `@demo remove-block-end`,
 }
 
+export const demoMarkupRegex = /\s*\/\/\s*@demo.*|{?\/.*@demo.*\/}?/gm
```

Comment:
As in the PR notes, this will not capture multiline block comments including @demo

Examples that are currently found:
- /* @demo */
- // @demo
- {/* @demo */}

Reply:
Could we leave this limitation as a comment in the codebase?

Reply:
Yeah good idea I'll add that.